.mailmap

```diff
@@ -204,6 +207,7 @@ Magnus Persson               <vilhelmp@gmail.com>
 Maneesh Yadav                <maneesh.yadav@sri.com>
 Mangala Gowri Krishnamoorthy <mangalagb@gmail.com>
 Marten van Kerkwijk          <mhvk@astro.utoronto.ca> <mhvk@swan.astro.utoronto.ca>
+Marten van Kerkwijk          <mhvk@astro.utoronto.ca> Marten H. van Kerkwijk <mhvk@astro.utoronto.ca>
```

Comment:
This looks weird.

Reply:
Why? Without that they would appear as 2 different persons.

Reply:
Oh, I thought this file only consolidate emails. Ignore me then.